package/MDAnalysis/coordinates/base.py

```diff
@@ -141,9 +141,15 @@
 from ..auxiliary.core import get_auxreader_for
 
 class Namespace(object):
-    """Empty class 
-    To allow storing attributes in new namespace """
-    pass
+    """Class to allow storing attributes in new namespace. """
```

Comment:
I think `pandas` gets this through special treatment by the `ipython` kernel. I think I remember @kain88-de finding this out when we were looking for something similar for `datreant`